projects/pipelines/testing/config.yaml

```diff
@@ -160,6 +160,17 @@ ci_presets:
     tests.pipelines.project_count: 5
     tests.pipelines.capture_extra_artifacts: false
 
+  gating:
```

Comment:
can you rename this maybe `gating_test`?

Reply:
Done!